### ISSUE TITLE
Any questions about upcoming update 0.7.4 or 0.8+

### ISSUE THREAD

**Issue by TDoodleF** [ShadowMario/FNF-PsychEngine#14150]
### What is your question?

Hello everyone and Psych engine team members
I'm taking about upcoming update 0.7.4 or 0.8+

•Stage Editor: A lot the people want the stage editor for make your own stages
•Modchart 2.0 lua: some modchart will be Kade Engine or yoshi crafter engine look like
•Character select: This feature you can put any custom playable characters
•Wavy Background: Lua supports allowing Background effects like different wavy effect like dave and bambi
•Mobile port support: Some mobile users can use mobile settings and touching buttons or hitbox
•Custom menu buttons: people can use custom menu like yoshi crafter engine
•Extra judgement: You can use custom judgements like os engine
•Custom pause feature: Allow to create your own pause menu with Lua
•Online/Multiplayer: You can play your friends around the world to play 1v1 with custom weeks or Custom songs
•Winning icon feature: you can add winning icon on your character

That's all
type in the comments if you want or not
Just a concept stuff
Let's wait a new update a couple months or weeks
See ya

**Comment by Stefan2008Git** [comment]
1. Stage Editor doesn't needed because you can simply learn lua really easy
2. Mochart editor doesn't needed too
3. Character editor already exist only on master editor menu
4. You can use the runtime shader for that shit (OS Engine fan :skull:)
5. There will never be any mobile support because @mcagabe19 and @KarimAkra already made one
6. Sir, for custom judgement counter you can add it yoursel by learning simply Haxe and no, this is the modified FNF base game engine
7. There will never be an online support!
8. The asshole support you mean for icons?

**Comment by mcagabe19** [comment]
1. no
2. no
3. character editor or ingame 8 or 9 ingame with debug build
4. FlxRuntimeShader and frag
5. we did mobile port and ShadowMario doesn't want
6. average os engine fan
7. psych online exists
8. go fuck yourself

**Comment by Stefan2008Git** [comment]
> 1. no
> 2. no
> 3. character editor or ingame 8 or 9 ingame with debug build
> 4. FlxRuntimeShader and frag
> 5. we did mobile port and ShadowMario doesn't want
> 6. average os engine fan
> 7. psych online exists
> 8. go fuck yourself

Yep, yep, yep

**Comment by Stefan2008Git** [comment]
> > 1. no
> > 2. no
> > 3. character editor or ingame 8 or 9 ingame with debug build
> > 4. FlxRuntimeShader and frag
> > 5. we did mobile port and ShadowMario doesn't want
> > 6. average os engine fan
> > 7. psych online exists
> > 8. go fuck yourself
> 
> Yep, yep, yep

Oh, btw did i forget Grafex Engine made first winning icon support after OS- Wait a minute, notweuz just stealed that too @mcagabe19

**Comment by TDoodleF** [comment]
Message for everyone:
The mobile port has already exist by people make mobile ports (You can port by a new update on FNF psych engine)
Thanks for the coolest question let's wait for the new update

**Comment by CobaltBar** [comment]
i hate this post so much

Learn how to code goddamnit, 90% of this can be done with lua or source
Most of these are already a no
Mobile Ports exist, i recommend MobilePorting/FNF-PsychEngine-Mobile
there are multiple online Psych Engine mods like UMM or Psych Online

tldr LEARN TO CODE

**Comment by Swapper1** [comment]
> > > 1. no
> > > 2. no
> > > 3. character editor or ingame 8 or 9 ingame with debug build
> > > 4. FlxRuntimeShader and frag
> > > 5. we did mobile port and ShadowMario doesn't want
> > > 6. average os engine fan
> > > 7. psych online exists
> > > 8. go fuck yourself
> > 
> > 
> > Yep, yep, yep
> 
> Oh, btw did i forget Grafex Engine made first winning icon support after OS- Wait a minute, notweuz just stealed that too @mcagabe19

You forgot about the custom menu buttons and custom pause feature.

**Comment by mcagabe19** [comment]
they edited so
custom pauss: lua exists

**Comment by BernardoGP4504** [comment]
1. Why? It's that hard to just edit an Lua file on notepad or whatever and press `Restart Song` on pause menu?
2. It's just better to learn it your own, like on YoshiCrafter Engine and Kade as you mentioned them
3. Just change either `player1`, `player2` or `gfVersion` in your chart file. An Character Editor already exists
4. No 💀. Also, just learn to apply the wiggle shader on the background, that's pretty much how it works
5. There already exists a [mobile port of Psych](https://github.com/MobilePorting/FNF-PsychEngine-Mobile)
6. Why? If you like OS Engine features so much, use it
7.  There's already some Psych forks that have multiplayer, best example being [Unnamed Multiplayer Mod](https://gamebanana.com/mods/431497)
8. Learn how to code, and once again: If you love OS Engine so much, use it and stop bothering others with it.


Kindly,
Bernard (BerGP)

**Comment by CobaltBar** [comment]
can't wait for shadowmario to see this and add 46 tags and close the post

that sounded sarcastic

**Comment by Swapper1** [comment]
I forgot to mention this earlier that the third one is actually wrong.
About character select, Minus mod, Dave & Bambi, and Retrospecter exists not character editor and/or anything from the Master Editor Menu. You can make your own by coding but for this Engine, no.

**Comment by Swapper1** [comment]
Mobile port support: That just the feature for Full Ass Game since we have Mobile Port already.
Custom menu buttons: Why do you want that feature?
Extra judgement: Camellia mod exists.
Online/Multiplayer: Same as the Mobile port, since Online Mod exists.
Winning icon feature: That's from FPS Plus.

**Comment by KarimAkra** [comment]
Yepi

**Comment by Stefan2008Git** [comment]
> Yepi

He listened to me Karim 🥲
